src/pages/resources/recipes/flyout-menu/index.md

```diff
@@ -0,0 +1,70 @@
+---
+keywords:
+  - Creative Cloud
+  - API Documentation
+  - UXP
+  - Plugins
+  - JavaScript
+  - ExtendScript
+  - SDK
+  - C++
+  - Scripting
+title: Flyout Menus in UXP InDesign
+description:
+---
+
+# Flyout Menus
```

Comment:
We want all recipes to follow a similar structure. Please use the template https://github.com/AdobeDocs/uxp-indesign/blob/main/src/pages/resources/recipes/_recipe_template.md.

Reply:
Comment Addressed.